src/materials/ShaderMaterial.js

```diff
@@ -95,6 +96,7 @@ ShaderMaterial.prototype.copy = function ( source ) {
 	this.vertexShader = source.vertexShader;
 
 	this.uniforms = cloneUniforms( source.uniforms );
+	this.uniformsGroups = source.uniformsGroups;
```

Comment:
We don't clone uniformsGroups array?

Reply:
I initially though it makes the material setup for UBOs easier if you can just clone a configured material for multiple objects. However, that only works if the `ShaderMaterial.copy()` ensures it does not clone the uniform groups, otherwise you end up with multiple UBOs you don't want.

However, I think now it's more consistent to perform a real clone and change the example code instead.